k8s/build-docker-image-from-source/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:20.04
+
+# For Chainese users, if there is something wrong, just comment it.
+ENV CC="clang-13"
+ENV CXX="clang++-13" 
+ENV PATH="$PATH:/var/apache-zookeeper-3.7.1-bin/bin:/var/RaftKeeper/bin"
+ENV RAFTKEEPER_DIR="/var/RaftKeeper"
+ENV TZ=Etc/UTC
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+
+
+RUN echo "deb [trusted=yes] http://mirrors.tuna.tsinghua.edu.cn/ubuntu/ focal main restricted universe multiverse" >> /etc/apt/sources.list \
```

Comment:
Tool long, separate it

Reply:
I intend to reduce the size of docker image, now I totally remove the file from the pull request.